flake.nix

```diff
@@ -656,6 +656,9 @@
               pre-commit-check
               solcDefault
               ;
+            extraPackages = with pkgs; [
+              nfpm
+            ];
```

Comment:
Should this not be in the cishell?

Reply:
It's in both, I execute it as well from my workspace (nix develop), but I'm not sure if this is the best place to be honestly.
@tolbrino can help on that